### PR TITLE
Fix a few typos

### PR DESCRIPTION
**Pull request by tatami4**

### CODE REVIEW

File: data/remnant/remnant missions.txt [hunk 1]
```diff
@@ -3106,10 +3106,10 @@ mission "Remnant: Cognizance 9"
 	to offer
 		has "Remnant: Cognizance 8: done"
 	on offer
-		log "People" "Gavriil" `A technician that works with Plume, and a talented astrobiologist. Gavriil is the project lead for the repurposed esquiline station.`
+		log "People" "Gavriil" `A technician that works with Plume, and a talented astrobiologist. Gavriil is the project lead for the repurposed Esquiline station.`
 		conversation
 			`You walk into Plume's lab and are greeted by a large stack of crates and equipment. Plume and another Remnant are gathered around the stack checking items off a list. "Ah, <first>! Good timing. We were just finishing the inventory of supplies that need to go on this delivery. He gestures to the engineer. "This is Gavriil, one of our best technologists and a skilled astrobiologist in his own right. He will be going with you."`
-			`	"It will be a pleasure to work with you," signs Gavrill, making a short bow. "We will be traveling to a research station on <planet>. Yes, it is a gas giant." He smiles at that. "I have to get some equipment setup and get the core systems online. There will be a few more loads of materials afterwards." He looks askance at you. "Shall we get started?"`
```
Comment (W1zrad) [inline data/remnant/remnant missions.txt:3112]: What's the change here as well?

**Review by Zitchas — APPROVED** [review]
Thanks for spotting these! Looks like a good set of fixes.

File: data/remnant/remnant missions.txt [hunk 2]
```diff
@@ -1236,7 +1236,7 @@ mission "Remnant: Catalytic Ramscoop"
 		payment 1390000
 		"remnant met taely" ++
 		conversation
-			`As the <ship> settles onto the pad closest to the research laboratory, you can see several people exiting an adjacent hangar with a trailer pulled by a camel. They quickly approach your landing pad and wait for you to complete your shutdown routines before boarding and starting to unload the ramscoops. Once they are done one of the comes up to you and hands you your payment of <payment>. "Thank you for the delivery, Captain. We are looking forward to seeing what they have done differently."`
```
[the remark this answers is not in the document]
Comment (Zitchas) [reply data/remnant/remnant missions.txt:1239]: The "the" should be a "them" in `Once they are done one of the comes up to you...`

[the remark this answers is not in the document]
Comment (Zitchas) [reply data/remnant/remnant missions.txt:1239]: I don't know why GitHub doesn't highlight changes like that.

[the remark this answers is not in the document]
Comment (W1zrad) [reply data/remnant/remnant missions.txt:1239]: Oh! Github's being mean and not pointing it out to me. Thanks!

**Review by W1zrad — APPROVED** [review]

File: data/remnant/remnant missions.txt (hunk 1 above)
Comment (Zitchas) [reply data/remnant/remnant missions.txt:3112]: Changing Gavrill's' name to match the way it is spelled in the log entry, Gavriil.

**Pull request merged**